numpy/f2py/crackfortran.py

```diff
@@ -3341,7 +3341,7 @@ def crack2fortran(block):
   and also be sure that the files do not contain programs without program statement).
 """, 0)
 
-    postlist = crackfortran(files, funcs)
+    postlist = crackfortran(files)
```

Comment:
The module documentation fot `crackfortran` also needs fixing.

Reply:
Updated.